src/sentry/api/issue_search.py

```diff
@@ -27,6 +27,7 @@ class IssueSearchVisitor(SearchVisitor):
         "assigned_to": ["assigned"],
         "bookmarked_by": ["bookmarks"],
         "subscribed_by": ["subscribed"],
+        "owner": ["owner"],
```

Comment:
This key won't show up in the autocompletes unless you also update `static/app/stores/tagStore.tsx`. The `getIssueAttributes()` will need to know about this field before you can wire up autocomplete to it. You'll likely also need to special case the value completion to use the team/memberlist.

Reply:
Thanks Mark - and David for tackling the frontend bits :-)